### PR TITLE
Add more Unique Vars

### PR DESCRIPTION
**Pull request by St19Galla**

### CODE REVIEW

File: formatter.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ var render = function(){
   final[lang]["translator"] = document.getElementById('translator').value.replace(/[\"]/g, "\\\"");
   var unique1 = document.getElementById('unique1').value.replace(/[\"]/g, "\\\"") + "\n" + "<br/><br/>";
   var unique2 = document.getElementById('unique2').value.replace(/[\"]/g, "\\\"") + "<br/>";
```
Comment (TheGrits) [inline formatter.js:26]: We're using two `<br/>`s for each line except the last one which only has one.

Comment (TheGrits) [inline formatter.js:26]: @St19Galla Can you make another PR to fix what I said above? Thanks!

**Comment by St19Galla** [comment]
@TheGrits Oh, I see. Will do!
